### DISCUSSION TITLE
A new name for neovim-flake!

### DISCUSSION THREAD

**Discussion by NotAShelf** [NotAShelf/nvf#246] [Polls]
Dear neovim-flake users,

It's been a while since we have made any changes to the main branch of the neovim-flake due to our ongoing work for the **v0.6** release, containing a whole bunch of quality of life improvements for both end-users and contributors - such as the generation of plugin setup options from Nix directly.

With v0.6 on the horizon, greatly refactoring this flake; I would like to take a look at renaming this project to something *less* generic. So far, I have **nvf** in mind - which makes a mildly funny acronym to both `NotAShelf's vim flake` and the ol' reliable `neovim-flake` but of course, your feedback is appreciated on this one. If you got anything in mind, please choose below. If you'd like to suggest your own name, please leave a comment.

Thank you for your continued support!
\- raf

**Comment by deviantsemicolon** [comment]
Nvf for NotaVimFlake

Because it's ackshually a Neovim flake

**Comment by FrothyMarrow** [comment]
Nixvim, to confuse people

**Reply by NotAShelf** [reply]
so true bestie

**Comment by ErrorNoInternet** [comment]
neoflake 🧌

**Comment by imnotpoz** [comment]
nvf goated
